docs/installing-cabal.md

```diff
@@ -36,18 +37,18 @@ didn't use the `bootstrap.sh` script, then make sure you run `cabal
 --version` to double check the version of the Cabal library that you
 need.
 
-*GHC 8.0.1 ships with Cabal-1.24.0.0 out of the box, so you may already
+*GHC 8.0.1 ships with Cabal-1.24.0.2 out of the box, so you may already
```

Comment:
I'd actually drop that section altogether. People outside Ambiata who are using GHC 8.0 should be using 8.0.2 anyway.